Creiwry/kitten_project
# Add devise model user

### What does this PR do?
PR adds users table generated with devise with additional attributes first_name, last_name, and city. Views, routes and models are auto-generated by devise

### How should this be manually tested?
run rails s
go to your server to the route users/sign_up
see if the sign-up page displays
